app/controllers/home/items_controller.rb

```diff
@@ -15,8 +15,9 @@ def show
   end
 
   def index
-    # @category = Category.find(params["category_id"])
-    # @items = @category.items
-    @items = Item.all
+    puts params
+    @categories = Category.all
+    @category = Category.find(params[:category_id])
+    @items = @category.items
   end
```

Comment:
Items of categories is all manged in home/categories_controller.rb, isn't it ? This method index in home/items_controller.rb is useless.

Reply:
Yes, I should have deleted it and the view after I puts the method in the show method for categories, I'll do it in my next pull request.